.github/workflows/lint-frontend.yml

```
@@ -0,0 +1,36 @@
+name: Lint Frontend
+
+on:
+  push:
+  pull_request:
+    types: [opened, synchronize]
+    paths:
+      - "web"
+
+env:
+  YARN_CACHE_FOLDER: .cache/yarn
+
+jobs:
+  lint:
+    name: lint frontend
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: web
+    steps:
+      - name: Check Out
+        uses: actions/checkout@v2
+      - name: Setup Node.js v16
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.x
```

Does this really work? I cannot find the syntax `16.X` on the official readme of setup-node action.
https://github.com/marketplace/actions/setup-node-js-environment

```suggestion
          node-version: '16'
```

---

Yes, it works.
setup-node downloads from `https://nodejs.org/dist` . And v16.x is released from that.
If you are worry, I change `16.x` to `16`.

FYI

https://github.com/actions/setup-node/blob/04c56d2f954f1e4c69436aa54cfef261a018f458/src/installer.ts#L266-L298

https://nodejs.org/dist/

---

I see. But, yeah, please change it. Just to be safe in the future, let's go the way that the official suggests.

---

I fixed it.